src/components/addpeer/WindowsTab.tsx

```diff
@@ -3,16 +3,18 @@ import React, {useState} from 'react';
 import { Button } from "antd";
 import TabSteps from "./TabSteps";
 import { StepCommand } from "./types"
+import {getConfig} from "../../config";
+const {latestVersion} = getConfig();
 
 export const WindowsTab = () => {
 
-    const releaseVersion = '0.6.2'
+    const releaseVersion = latestVersion ? latestVersion : "v0.6.3"
```

Comment:
Codacy has a fix for the issue: [Replace `··const·releaseVersion·=·latestVersion·?·latestVersion·:·"v0.6.3"` with `const·releaseVersion·=·latestVersion·?·latestVersion·:·"v0.6.3";`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9561089)
```suggestion
  const releaseVersion = latestVersion ? latestVersion : "v0.6.3";
```